acceptance/tests/pxp-module-puppet/run_puppet_killed_puppet.rb

```diff
@@ -75,6 +70,10 @@
       else
         on agent, "kill -9 #{pid}"
       end
+
+      # Also halt the sleep process here, as on Windows the parent process won't exit while the child process
+      # has an open handle shared with it (the stdout pipe).
+      stop_sleep_process(agents)
```

Comment:
`agents` -> `agent`
otherwise there is an attempt to kill the sleep process on *all* agents on every iteration through the `agents` array